src/Files.App/ViewModels/UserControls/ToolbarViewModel.cs

```diff
@@ -461,6 +479,7 @@ public void PathItemSeparator_DataContextChanged(FrameworkElement sender, DataCo
 
 		public void PathboxItemFlyout_Opened(object sender, object e)
 		{
+			IsCommandPaletteOpen = false;
```

Comment:
I don't think this modification is necessary.

Reply:
You are correct. Thanks